<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="centralwidget_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="ViewerWidget" name="viewer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_open_model"/>
    <addaction name="action_open_animation"/>
    <addaction name="separator"/>
    <addaction name="action_save_model"/>
    <addaction name="action_save_model_as"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_window"/>
  </widget>
  <widget class="QDockWidget" name="dock_explorer">
   <property name="windowTitle">
    <string>Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dock_explorer_contents">
    <layout class="QVBoxLayout" name="dock_explorer_contents_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="ExplorerWidget" name="explorer">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_preview">
   <property name="windowTitle">
    <string>Preview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dock_preview_contents">
    <layout class="QHBoxLayout" name="dock_preview_contents_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="PreviewWidget" name="preview" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_view_settings">
   <property name="windowTitle">
    <string>View Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dock_view_settings_contents">
    <layout class="QVBoxLayout" name="dock_view_settings_contents_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scroll_area_view_settings">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="ViewSettingsForm" name="view_settings">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>77</width>
          <height>548</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_texture">
   <property name="windowTitle">
    <string>Texture</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dock_texture_contents">
    <layout class="QVBoxLayout" name="dock_texture_contents_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scroll_area_texture">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="TextureForm" name="texture">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>254</width>
          <height>134</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_open_model">
   <property name="text">
    <string>&amp;Open Model...</string>
   </property>
  </action>
  <action name="action_open_animation">
   <property name="text">
    <string>Open Animation...</string>
   </property>
  </action>
  <action name="action_save_model">
   <property name="text">
    <string>&amp;Save Model</string>
   </property>
  </action>
  <action name="action_save_model_as">
   <property name="text">
    <string>Save Model &amp;As...</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_texture_export">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="action_texture_replace">
   <property name="text">
    <string>&amp;Replace</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExplorerWidget</class>
   <extends>QTreeWidget</extends>
   <header>widgets.explorer_widget</header>
  </customwidget>
  <customwidget>
   <class>ViewerWidget</class>
   <extends>QWidget</extends>
   <header>widgets.viewer_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewSettingsForm</class>
   <extends>QWidget</extends>
   <header>widgets.view_settings_form</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextureForm</class>
   <extends>QWidget</extends>
   <header>widgets.texture_form</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets.preview_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
